api/api/settings.py

```diff
@@ -29,6 +29,7 @@
     'localhost',
     '127.0.0.1',
     'api.portfolio.isidel.com',
+    'http://0.0.0.0:8000/'
```

Comment:
Not important this time but in general this should be in a separate commit from the sorting stuff, as it's unrelated code, e.g. a separate commit titled 

[API] Add 0.0.0.0 host as an allowed dev host

Also ALLOWED_HOSTS should be in the format with just 0.0.0.0, not with the http and port.

Anyway do you even need this? Was this when you were trying to develop on google cloud?